File: service/TFC.Application.DTO/Exercise/AddExercise/AddExerciseRequest.cs
namespace TFC.Application.DTO.Exercise.AddExercise
{
    public class AddExerciseRequest
    {
        public List<string> ExercisesProgres { get; set; } = new List<string>();
        public string UserEmail { get; set; } = string.Empty;
        public int RoutineId { get; set; }
        public string DayName { get; set; } = string.Empty;
    }
}

File: tfc_gym_app/lib/data/models/dto/exercise/add_exercise/add_exercise_request.dart
class AddExerciseRequest {
  final int routineId;
  final String dayName;
  String email;
  final String exerciseName;

  AddExerciseRequest({
    required this.dayName,
    required this.routineId,
    required this.exerciseName,
    required this.email,
  });
}